_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Self-host maps server
+nav_order: 6
+---
+
+# Self-host maps server
+
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service on air-gapped environments. The server provides a map manifest url with map tiles and vectors, a map tiles url, and a map vectors url that are compatible with OpenSearch.
```

Comment:
AWS appears to capitalize URL

Reply:
Yeah - URL should be caps both instances

Reply:
revised